### PR TITLE
Adding circular progress bar label

### PR DESCRIPTION
**Pull request by adicicurel**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6192

File: src/Themes/businessDashboard/docs/businessDashboard.story.js [hunk 1]
```diff
@@ -103,6 +103,10 @@ import { theme } from 'wix-style-react/themes/businessDashboard';
   ],
 };
 
+export function createExample() {
```
Comment (shlomitc) [inline src/Themes/businessDashboard/docs/businessDashboard.story.js:106]: looks like a mistake

File: src/CircularProgressBar/docs/index.story.js [hunk 2]
```diff
@@ -89,6 +90,12 @@ export default {
             source: examples.progressIndication,
           }),
 
+          code({
+            title: 'Label',
+            description: 'Displaying a label in percentage.',
```
Comment (shlomitc) [inline src/CircularProgressBar/docs/index.story.js:95]:
```suggestion
            description: 'Displaying a textual label',
```

File: src/CircularProgressBar/docs/examples.js [hunk 3]
```diff
@@ -46,3 +46,17 @@ export const errors = `<Layout>
...
+<Cell>
+  <CircularProgressBar error value={20} />
+</Cell>
+<Cell>
+  <Box height={54} width={54} backgroundColor="D10">
+     <CircularProgressBar light error value={20} />
+  </Box>
+</Cell>
+  <Cell>
+  <CircularProgressBar label="1/5" showProgressIndication error errorMessage="some error" value={20} />
```
Comment (shlomitc) [inline src/CircularProgressBar/docs/examples.js:60]: I think you can keep only this and remove the other examples

**Review by shlomitc — CHANGES_REQUESTED** [review]
Looks great!
one minor request for change + please add a visual test for the new label prop

File: src/CircularProgressBar/docs/examples.js (hunk 3 above)
Comment (adicicurel) [reply src/CircularProgressBar/docs/examples.js:60]: i will remove the percentage, but the success is still valid.
Thanks

File: src/Themes/businessDashboard/docs/businessDashboard.story.js (hunk 1 above)
Comment (adicicurel) [reply src/Themes/businessDashboard/docs/businessDashboard.story.js:106]: removed

**Comment by adicicurel** [comment]
> Looks great!
> one minor request for change + please add a visual test for the new label prop

done

**Pull request merged**